### ISSUE TITLE
Make floatUrl work with keyboard shortcut

### ISSUE THREAD

**Issue by SantanuDatta** [zen-browser/theme-components#5]
what I am trying to say is for now lets say if I use ctrl+t it opens a new tab but the position of that new tab remains in the top instead of opening the floatUrl in the middle, I want the floaturl to show up when I use the shortcut like ctrl+t instead of clicking the tab to see the floaturl just like arc browser.

**Comment by 22ispencer** [comment]
I'm just starting to use zen, but I originally noticed this too, but when you start typing it focuses the bar and goes to the expected position, maybe it'd be possible to add a setting to focus by default with C-t

**Comment by SantanuDatta** [comment]
> I'm just starting to use zen, but I originally noticed this too, but when you start typing it focuses the bar and goes to the expected position, maybe it'd be possible to add a setting to focus by default with C-t

Yeah, hopefully dev will look at this issue or maybe need to submit it in the main repository.

**Comment by KhanKudo** [comment]
It does work, when you use Ctrl+L, however this changes the current tab rather than opening a new one. I absolutely agree, that Ctrl+T should have an option to also open floating URL immediately

**Comment by Tealk** [comment]
> It does work, when you use Ctrl+L

For me, the floating bar only appears when I start typing in the URL bar, otherwise it just stays at the top of the window.
